dropbox-sdk-dotnet/Dropbox.Api/DropboxClient.cs

```diff
@@ -186,6 +186,16 @@ public DropboxClient WithPathRoot(PathRoot pathRoot)
             return new DropboxClient(this.requestHandler.WithPathRoot(pathRoot));
         }
 
+        /// <summary>
+        /// Refreshes access token regardless of if existing token is expired
```

Comment:
Why do we need to refresh the token even when it is not expired? To acquire more scopes?

Reply:
This is needed for downscoping.  New/more scopes would require a new oauth flow